code/modules/mob/mob.dm

```diff
@@ -1188,13 +1188,15 @@
 	return mob_location.get_lumcount() > light_amount
 
 /// Can this mob read
-/mob/proc/can_read(obj/O, check_for_light = TRUE)
-	if(!is_literate())
-		to_chat(src, span_warning("You try to read [O], but can't comprehend any of it."))
+/mob/proc/can_read(atom/viewed_atom, check_for_light = TRUE, check_literacy = TRUE, chat_messages = TRUE)
```

Comment:
Remove the `check_literacy` and `chat_messages` args since they aren't needed.  `check_literacy` bypasses the entire proc and makes all the code redundant.

I was the author of the new literacy code and illiterate quirk so I refactored a lot of this code and I want to keep it clean and organized.

Reply:
Check literacy doesn't bypass the entire proc?????